coalahtml/_coalahtml/app/controllers/files.js

```diff
@@ -2,37 +2,51 @@
 
 angular.module('coalaHtmlApp')
   .controller('FilesCtrl', function ($scope, $http, $routeParams) {
+    $http.get("data/Constants.json").success(function(content) {
+      $scope.Constants = content;
+      $http.get($scope.Constants.roothome).success(function(content) {
```

Comment:
It would be just slightly easier to understand with meaningful names. That's the only thing